submissions/Ant-C-tech/memory-pair-game/js/app.js

```diff
@@ -0,0 +1,278 @@
+'use strict'
+
+import hiragana from './hiragana.js'
+
+
+const BODY = document.querySelector('.body')
+const MAIN = document.querySelector('#main')
+const HEADER = document.querySelector('#header')
+const DESCRIPT = document.querySelector('#descript')
+
+const CARDS_NUMBER = 12;
+const ANIMATE_CSS_CLASS = 'animate__animated'
+const ANIMATE_DELAY = 'animate__delay-3s'
+const DESCRIPT_HIDE_ANIMATION = 'animate__zoomOut'
+const DESCRIPT_SHOW_ANIMATION = 'animate__zoomIn'
+const CARDS_HIDE_ANIMATION = 'animate__rotateOut'
+
+let TASK = []
+let attempts = 0
+    //Music
+const GAMEPLAY_AUDIO = new Audio('./audio/Japanese_Countryside.mp3')
+GAMEPLAY_AUDIO.loop = true
+GAMEPLAY_AUDIO.volume = 0.3
+
+
+initApp()
+
+
+function initApp() {
+    setTask()
+    const timeout = setTimeout(() => {
+        changeContent(createGreeting())
+        clearTimeout(timeout)
+    }, 1500);
+}
+
+function setTask() {
+    TASK.push(hiragana[_getRandomIntInclusive(0, hiragana.length - 1)])
+    while (TASK.length < CARDS_NUMBER / 2) {
+        const currentItem = hiragana[_getRandomIntInclusive(0, hiragana.length - 1)]
+        let flag = false
+        for (const item of TASK) {
+            if (item.jap === currentItem.jap) {
+                flag = true
+            }
+        }
+        if (flag === false) {
+            TASK.push(currentItem)
+        }
+    }
+    TASK = TASK.concat(TASK)
+}
+
+function createGreeting() {
+    const greeting = document.createElement('h3')
+    greeting.textContent = 'Kottans present'
```

Comment:
I'd recommend storing such strings in constants. It can help to edit in one place and not to find it in the functions.